### ISSUE TITLE
Normal tear production on Schirmer I test

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11646]
**Preferred term label:**
Normal tear production on Schirmer I test

**Synonyms:**
* Normal lacrimation on Schirmer I test
* Normal tear flow on Schirmer I test
* Schirmer I test greater than 5 mm in 5 minutes

**Definition (free text, please give PubMed ID):**
The presence of normal tear flow as clinically indicated by a Schirmer I test result of greater than 5 millimeters of wetting on the standard filter paper strip in a 5-minute period. PMID: 32298795

**Parent term (use hpo.jax.org/app):**
Lacrimation abnormality (HP:0000632) *(Note: While HPO typically catalogs abnormal clinical features, this term is proposed to allow the clinical representation of a preserved or normal finding in phenotyping datasets).*

**Your nano-attribution (ORCID):**
0000-0001-9042-0508

**Comment by pnrobinson** [comment]
@daboe01 The HPO does not have terms for "normal". Instead, we would use the "Lacrimation abnormality" term with EXCLUDED modifier. This can be done easily with the phenopacket schema or by adding a column in a database/excel file etc.

**Comment by daboe01** [comment]
excellent! thank you

**Comment by pnrobinson** [comment]
@daboe01 thanks, closing!
